docs/book/driver-deployment/prerequisites.md

```diff
@@ -0,0 +1,156 @@
+- [Virtual Machine Configuration](#vm_configuration)
+- [vSphere Cloud Provider Interface (CPI)](#vsphere_cpi)
+
+## Compatible vSphere and ESXi versions <a id="compatible_vsphere_esxi_versions"></a>
+
+VMware's Cloud Native Storage (CNS) solution is delivered on vSphere version 6.7U3 and above. Volume driver interacts with CNS and hence only connects to vSphere 6.7U3 and above. When upgrading an older version of vSphere to 6.7U3 or above, make sure to upgrade the individual ESXi hosts that are part of the cluster to 6.7U3 or above as well. Note that the vSphere versions and ESXi versions should match.
+
+## vSphere Roles and Privileges <a id="roles_and_privileges"></a>
+
+The vSphere user for CSI driver requires set of privileges to perform Cloud Native Storage operations.
+
+To know how to create and assign role, refer the [vSphere Security documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-41E5E52E-A95B-4E81-9724-6AD6800BEF78.html).
+
+Following roles needs to created with sets of permissions.
+
+| Role                    | Privileges for the role | Required on                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+|-------------------------|-------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CNS-DATASTORE           | ![ROLE-CNS-DATASTORE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/ROLE-CNS-DATASTORE.png)<br><pre lang="bash">govc role.ls CNS-DATASTORE<br>Datastore.FileManagement<br>System.Anonymous<br>System.Read<br>System.View<br></pre>| Shared datastores where persistent volumes needs to be provisioned..                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
s/Shared datastores where persistent volumes needs to be provisioned.. /Shared datastores where persistent volumes need to be provisioned.

Reply:
done